internal/manifest/list.go

```diff
@@ -65,6 +69,22 @@ type ListUpdate struct {
 	MediaType string
 }
 
+type instanceOperation int
```

Comment:
Absolutely non-blocking: In an internal package, this can be public — that would allow callers to store that value in a variable (and in particular, it could allow `c/image/copy` to use this enum, instead of inventing its own, to represent “that kind of an update will need to happen” when creating the copy plan, when the full `ListEdit` value can not yet be constructed.

But we can change that when/if a user is added.